cmd/prom-migrator/main.go

```diff
@@ -40,17 +41,23 @@ type config struct {
 	progressMetricName string
 	progressMetricURL  string
 	progressEnabled    bool
+	versionPrint       bool
 	readerAuth         utils.Auth
 	writerAuth         utils.Auth
 }
 
 func main() {
 	conf := new(config)
 	parseFlags(conf, os.Args[1:])
+	if conf.versionPrint {
+		fmt.Println("Version: ", version.Version)
+		os.Exit(0)
+	}
```

Comment:
Can you please make this like: https://github.com/timescale/promscale/blob/5de3a5ce6c634508bc524e32e9cad3f7470d586d/pkg/runner/args.go#L14
That will enable to easily expand the args in future.

Reply:
Roger that. I just changed it to match promscale CLI.